semantics/SEMANTICS.md

```diff
@@ -1,4051 +1,4051 @@
-  </tr>
-  <tr>
-    <td>ns is [1, 2, 3]<br/>for n in ns<br/>&emsp;&emsp;print n</td>
-    <td>ns = [1, 2, 3]<br/>for n in ns:<br/>&emsp;&emsp;print(f'{n}')</td> 
-  </tr>
-  <tr>
-    <td>for a in range 2 to 4:<br/>&emsp;&emsp;a is a + 2</td>
-    <td>step = 1 if int(2) &lt; int(4) else -1<br/>for a in range(int(2), int(4) + step, step):<br/>&emsp;&emsp;a = a + 2</td>
-  </tr>
-  <tr>
-    <td>a is 0<br/>while a < 3:<br>&emsp;&emsp;a is a + 1</td>
-    <td>a = 0<br/>while str(a).zfill(100)&lt;str(3).zfill(100):<br/>&emsp;&emsp;a = a + 1</td>
-  </tr>
-</tbody>
-</table>
-
-
-#### Incorrect Programs
-
-<table>
-<thead>
-  <tr>
-    <th>Hedy</th>
-    <th>Exception</th>
-  </tr>
-</thead>
-<tbody>
-  <tr>
-    <td>print</td>
-    <td>Incomplete Exception</td>
-  </tr>
-  <tr>
-    <td>print'x'</td>
-    <td>Invalid command</td>
-  </tr>
-  <tr>
-    <td>print 1,0 + 1</td>
-    <td>Invalid Command (TODO: this could be improved)</td>
-  </tr>
-  <tr>
-    <td>ask</td>
-    <td>Incomplete Exception</td>
-  </tr>
-    <tr>
-    <td>(ask | print) (any text without quotes around it)</td>
-    <td>Unquoted Text Exception</td>
-  </tr>
-  <tr>
-    <td>(not (print | ask)) (any text) (not is) (any text)?</td>
-    <td>Invalid Exception</td>
-  </tr>
-  <tr>
-    <td>_SPACE (print | ask) (any text)?</td>
-    <td>Invalid space</td>
-  </tr>
-  <tr>
-    <td>_SPACE (any text) is (any text)?</td>
-    <td>Invalid space</td>
-  </tr>
-  <tr>
-    <td>a is [1, 2, 3]<br/>answer is ask 'Is the number in ' a</td>
-    <td>Invalid Argument Type Exception</td>
-  </tr>
-  <tr>
-    <td>a is [1, 2, 3]<br/>f is 4<br/>(add | remove) a (to | from) f</td>
-    <td>Invalid Argument Type Exception</td>
-  </tr>
-  <tr>
-    <td>items is [1, 2, 3]<br/>print items</td>
-    <td>Invalid Argument Type Exception</td>
-  </tr>
-  <tr>
-    <td>a is 'test'<br/>print a + 1</td>
-    <td>Invalid Argument Type Exception</td>
-  </tr>
-  <tr>
-    <td>a is 'text'<br/>one is 1<br/>if one in a<br/>&emsp;&emsp;print 'found!'</td>
-    <td>Invalid Argument Type Exception</td>
-  </tr>
-  <tr>
-    <td>name is [1, 2]<br/>if '1' is name<br/>&emsp;&emsp;print 'found'</td>
-    <td>Invalid Argument Type Exception</td>
-  </tr>
-  <tr>
-    <td>if '1' < 3:<br/>&emsp;&emsp;print 'true'</td>
-    <td>Invalid Argument Type Exception</td>
-  </tr>
-  <tr>
-    <td>for a in range 1 to 10:<br/>print 'hello!'</td>
-    <td>No Indentation Exception</td>
-  </tr>
-  <tr>
-    <td>x is 1, 2, 3</td>
-    <td>Invalid Exception (?)</td>
-  </tr>
-  <tr>
-    <td>x is [one, two, three]</td>
-    <td>Unquoted text Exception</td>
-  </tr>
-  <tr>
-    <td>(if | for | while) (boolean) (newline without ':' first) </td>
-    <td>Invalid Exception</td>
-  </tr>
-  <tr>
-    <td>(any lines with code without `if`) elif (boolean):</td>
-    <td>Invalid Exception</td>
-  </tr>
-</tbody>
-</table>
+# Semantics
```

Comment:
not sure why it shows as the entire file has changed.  The change here should just be the removal of `.zfill(100)`

Reply:
This probably has to do with line endings, but it is not such a relevant file to keep the git blame history of, don't worry!